src/modules/launcher/Plugins/Community.PowerToys.Run.Plugin.VSCodeWorkspaces/WorkspacesHelper/VSCodeWorkspacesApi.cs

```diff
@@ -70,7 +73,7 @@ public List<VSCodeWorkspace> Workspaces
                         {
                             VSCodeStorageFile vscodeStorageFile = JsonSerializer.Deserialize<VSCodeStorageFile>(fileContent);
 
-                            if (vscodeStorageFile != null)
+                            if (vscodeStorageFile != null && vscodeStorageFile.OpenedPathsList != null)
```

Comment:
When VS Code v1.64 is out and installed locally (updated current version) this file will still exist & contain `OpenedPathsList`? In that case new workspaces stored in new file won't be picked up?

Also, on VS Code update, will workspaces from old storage.json file be copied to new file or not ?

Reply:
the storage.json file will exist but the OpenedPathsList will not. 
When vscode updates the workspaces from the storage.json will be moved to Backup/workspaces.json by vscode as far as i know.